analytics/docs/swagger.yaml

```diff
@@ -185,4 +185,4 @@ definitions:
 
 externalDocs:
   description: "The Creative Commons search API"
-  url: "https://api.creativecommons.engineering"
+  url: "https://api.openverse.engineering"
```

Comment:
Should we go ahead and change the "Creative Commons search API" in the description, too?